options/posix/include/dlfcn.h

```diff
@@ -37,11 +39,26 @@ typedef struct {
 	void *dli_saddr;
 } Dl_info;
 
+struct dl_find_object {
+	unsigned long long dlfo_flags;
+	void *dlfo_map_start;
+	void *dlfo_map_end;
+	struct link_map *dlfo_link_map;
+	void *dlfo_eh_frame;
+	unsigned long long __dlfo_unused[7];
```

Comment:
glibc seems to have two optional members (hidden behind DLFO_STRUCT_HAS_EH_DBASE and DLFO_STRUCT_HAS_EH_COUNT respectively), but those don't get used on my system apparently.